.github/workflows/zizmor.yml

```diff
@@ -0,0 +1,23 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
+
+on:
+  pull_request:
+    branches: ["**"]
+
+permissions: {}
+
+jobs:
+  zizmor:
+    name: zizmor latest via PyPI
```

Comment:
Does this mean that `zimor` only gives github actions attack protection until the project on PyPI gets pwned?